shared/types/workspace.ts

```diff
@@ -0,0 +1,15 @@
+import { Opaque } from 'type-fest'
+import { FormId } from './form'
+import { UserId } from './user'
+
+export type WorkspaceId = Opaque<string, 'WorkspaceId'>
+
+export type Workspace = {
+  _id: WorkspaceId
+  title: string
+  count: number
```

Comment:
what's this `count` for? doesn't seem to be in the design doc, in fact the doc says that the count is not needed

Reply:
ah, I see it becomes a virtual later on